### PR TITLE
Exams by Mariia Sribranets IM-33

### PR DESCRIPTION
**Pull request by MariiaSribranets**

### CODE REVIEW

File: Tasks/between.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+  if (initialIndex === -1) return "";
+
+  str = str.substring(initialIndex + prefix.length);
+  initialIndex = str.indexOf(suffix);
+
+  if (initialIndex === -1) return "";
+  
+  str = str.substring(0, initialIndex);
+  
   return str;
```
Comment (tshemsedinov) [inline Tasks/between.js:17]: Do not change incoming args

File: Tasks/compare.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-  let a = Object.keys(first_values);
-  let b = Object.keys(second_values);
-  if (a.join('-') !== b.join('-')) return false;
-  let e = true;
-  for (c of a) {
-    if (first_values[c] === second_values[c]) e = e && true;
-    else {
-      e = e && false;
-    }
+let compare = (first_values, second_values) => {
```
Comment (tshemsedinov) [inline Tasks/compare.js:4]: Respect camelCase naming conventions

File: Tasks/drop.js [hunk 3]
```diff
@@ -1,24 +1,14 @@
...
-    },
-    ['uno', 'due', 'tre']
-  );
-  T = D;
-  return D;
+const dropKeys = (dictionary, ...deleteKeys) => {
+  const keys = Object.keys(dictionary);
+  
+  for(const key of keys){
+    if (deleteKeys.includes(key)) delete dictionary[key];
```
Comment (tshemsedinov) [inline Tasks/drop.js:8]: impediment without delete

File: Tasks/between.js (hunk 1 above)
Comment (MariiaSribranets) [reply Tasks/between.js:17]: Understood. Thank you for your comment

File: Tasks/compare.js (hunk 2 above)
Comment (MariiaSribranets) [reply Tasks/compare.js:4]: Got it. Thank you for your comment

**Pull request closed without merging**